### PR TITLE
feat: add coolblue store

### PR DESCRIPTION
**Pull request by FeikoWielsma**
### Description
Adds the Coolblue store since they seem to have stock sometimes since this morning.

### Testing
I changed the URL into a 2060 to test, it seemed to work. I'm new to Javascript though (and pull requests in general), so let me know if I did anything weird!

### CODE REVIEW

File: src/store/model/coolblue.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import {Store} from './store';
+
+export const Coolblue: Store = {
+	labels: {
+		inStock: {
+			container: '.product-order',
+			text: ['bestel snel', 'Morgen in huis']
```
Comment (jef) [inline src/store/model/coolblue.ts:7]: You'll want to lowercase these as we do comparisons with all lowercase.
```suggestion
			text: ['bestel snel', 'morgen in huis']
```

File: src/store/model/coolblue.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		},
+		outOfStock: {
+			container: '.product-order',
+			text: ['Binnenkort leverbaar', 'Tijdelijk uitverkocht']
```
Comment (jef) [inline src/store/model/coolblue.ts:11]:
```suggestion
			text: ['binnenkort leverbaar', 'tijdelijk uitverkocht']
```

Same here

File: src/store/model/coolblue.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			brand: 'test:brand',
+			model: 'test:model',
+			series: 'test:series',
+			url: 'https://www.coolblue.nl/product/826844/'
+		},
+		{
+			brand: 'gigabyte',
+			model: 'eagle oc',
+			series: '3080',
+			url: 'https://www.coolblue.nl/product/868737/gigabyte-geforce-rtx-3080-eagle-oc-10g.html'
```
Comment (jef) [inline src/store/model/coolblue.ts:25]:
```suggestion
			url: 'https://www.coolblue.nl/product/868737'
```

Do you mind normalizing these URLs like the test model?

Thanks!

**Review by jef — CHANGES_REQUESTED** [review]
Just a few things. Looks good otherwise!

File: src/store/model/coolblue.ts (hunk 3 above)
Comment (FeikoWielsma) [reply src/store/model/coolblue.ts]: Done! (I think, no idea how to work with Github, I usually just use Git by itself)

**Review by jef — APPROVED** [review]

**Comment by jef** [comment]
Thank you!

**Pull request merged**